microsoft/testsuites/acctests/sgxtest.py

```diff
@@ -0,0 +1,217 @@
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT license.
+
+import os
+import pytest
+import subprocess
+
+from typing import Any
+from pathlib import Path
+from typing import Any
+from assertpy import assert_that
+
+from lisa import (
+    CustomScript,
+    CustomScriptBuilder,
+    Logger,
+    Node,
+    TestCaseMetadata,
+    TestSuite,
+    TestSuiteMetadata,
+    simple_requirement,
+)
+from lisa.operating_system import Posix
+from lisa.tools import Echo, Uname
+
+########### HELPER METHODS ###################
+
+@staticmethod
```

Comment:
It doesn't need to specify static for method in module. It's only needed for methods in a class.